distribution/android/app/build.gradle

```diff
@@ -100,3 +130,14 @@ tasks.whenTaskAdded { mergeAssetsTask ->
     mergeAssetsTask.dependsOn copyLanguageAssets
     mergeAssetsTask.dependsOn copySegmentFile
 }
+
+project.afterEvaluate {
+    // Download the licenses when (actually: just before) the
```

Comment:
Ah yeah. It's all quite a mess at the moment. Would be better to wait with your checks until it actually compiles and cleaned up the other files.